R/tableofdifferences.R

```diff
@@ -100,6 +106,7 @@
 
 TableOfDifferences <- function(table1,
                                table2,
+                               output = c("widget", "qtable")[1],
```

Comment:
Typically, instead of using `[1]`, `match.arg` is used. See the first example [here|https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/match.arg].